packages/sit-onyx/src/components/OnyxSelect/types.ts

```diff
@@ -70,9 +71,17 @@ export type SelectModelValueProps<TValue extends SelectOptionValue> =
 export type OnyxSelectProps<TValue extends SelectOptionValue = SelectOptionValue> = DensityProp &
   SelectModelValueProps<TValue> &
   SelectSearchProps &
-  Omit<OnyxSelectInputProps, "density" | "modelValue" | "showFocus"> &
+  Omit<OnyxSelectInputProps, "density" | "modelValue" | "showFocus" | "disabled" | "readonly"> &
   AutofocusProp &
   Pick<BaseSelectOption, "truncation"> & {
+    /**
+     * Whether the select should be disabled.
+     */
+    disabled?: FormInjected<boolean>;
+    /**
+     * Whether the select should be readonly.
+     */
+    readonly?: boolean;
```

Comment:
Why don't we just not `Omit<>` readonly here? Seems to be the same

Reply:
done